src/meta/src/stream/stream_manager.rs

```diff
@@ -642,6 +647,23 @@ impl GlobalStreamManager {
         state_table_ids: Vec<risingwave_meta_model::TableId>,
         fragment_ids: HashSet<FragmentId>,
     ) {
+        // TODO(august): This is a workaround for canceling SITT via drop, remove it after refactoring SITT.
+        for &job_id in &streaming_job_ids {
+            if self
+                .creating_job_info
+                .check_job_exists(TableId::new(job_id as _))
+                .await
+            {
+                tracing::info!(
+                    ?job_id,
+                    "streaming job is creating, cancel it with drop directly"
+                );
+                self.metadata_manager
```

Comment:
Should we also remove the entry in `self.creating_job_info`?

Reply:
The entry will be deleted by the create handler upon receiving this error.